h/static/scripts/polyfills.js

```diff
@@ -0,0 +1,15 @@
+// ES2015 polyfills
+require('core-js/es6/promise');
+require('core-js/fn/object/assign');
+
+// URL constructor, required by IE 10+,
+// Microsoft Edge.
+window.URL = require('js-polyfills/url').URL;
```

Comment:
Indeed. The polyfill's own feature detection unfortunately does not work when loaded as a CommonJS module because it relies on `this === window`. However, we can do our own feature detection trivially and I've implemented that. Of the different levels of support listed in the comment you linked to, the important thing we need is the `URL` constructor.

I've also corrected the comment because it turns out that current versions of Edge do support the URL constructor.